pygeos/constructive.py

```diff
@@ -285,6 +286,24 @@ def extract_unique_points(geometry, **kwargs):
     return lib.extract_unique_points(geometry, **kwargs)
 
 
+def normalize(geometry, **kwargs):
+    """Converts Geometry to normal form (or canonical form).
```

Comment:
Could you expand on the docstring?  I don't think that the terms normal or canonical are sufficiently well-understood by all users to know what this means (I don't know what they mean).  In particular, how it impacts the ordering of geometries within a collection / Multi or ordering of inner rings within a polygon.

Reply:
Yeah, this is just a copy of the GEOS docstring for now. 

I also don't know the details, but it might suffice to say that the coordinates / parts of Multi geometries / innter rings of a Polygon get ordered *consistently*?